constants/auth.ts

```diff
@@ -0,0 +1,3 @@
+export const TOKEN = {
+  access: "access_token",
```

Comment:
상수 컨벤션 + 문자열은 카멜케이스 요렇게 표현해주면 좋을 것 같아요~
```suggestion
  ACCESS_TOKEN: "accessToken",
```